generated_for_cmake/upb/json/parser.c

```diff
@@ -735,7 +735,7 @@ static bool multipart_text(upb_json_parser *p, const char *buf, size_t len,
 /* Note: this invalidates the accumulate buffer!  Call only after reading its
  * contents. */
 static void multipart_end(upb_json_parser *p) {
-  UPB_ASSERT(p->multipart_state != MULTIPART_INACTIVE);
+  /* UPB_ASSERT(p->multipart_state != MULTIPART_INACTIVE); */
```

Comment:
Remove?

Reply:
I'm leaving it because it was false sometimes, so it causes a run with assertions to fail. But I want to preserve it because it was theoretically supposed to be true. I'll add a comment to explain.